### ISSUE TITLE
Preserve state of hiding subject notes in Recents

### ISSUE THREAD

**Issue by DarrenMcRoy** [zooniverse/Panoptes-Front-End#1869]
I've noticed that whenever you return to a Recents page on Talk, the "subject notes" is always clicked, so you always have to manually unclick it if you just want to see non-subject discussions.

Could we perhaps preserve the state of not displaying those, either on a per-Talk basis or across all of Talk?

**Comment by parrish** [comment]
PR incoming....

**Comment by mschwamb** [comment]
@parrish this caused something buggy. The check box for showing the subject notes is checked by default, but wow subject notes don't show unless I deliberately uncheck and check the box

Going there on login without touching the check box and leaving it as it was checked

![fireshot capture 13 - comet hunters __ - https___www zooniverse org_project](https://cloud.githubusercontent.com/assets/753325/14561731/fad97d28-034a-11e6-9e6f-d4da0967144c.png)

now unchecking and checking it

![fireshot capture 14 - comet hunters __ - https___www zooniverse org_project](https://cloud.githubusercontent.com/assets/753325/14561733/06c9f7fc-034b-11e6-8bc4-53a0911cb1b0.png)

Basically this appears after unchecking and checking - 

<img width="986" alt="screen shot 2016-04-15 at 8 45 54 pm" src="https://cloud.githubusercontent.com/assets/753325/14561749/1a7d9128-034b-11e6-84fc-ae9e5754af3d.png">

So it looks like this change is defaulting to not showing the notes posts even though it is checked

**Comment by mschwamb** [comment]
Seems to be happening logged in and non-logged in. I've only tried with Firefox on Mac
